src/components/ratings.tsx

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018 Ultimaker B.V.
+import * as React from 'react';
+import {IconColor, IconWrapperProps} from './icons/icon_wrapper'
+import _ = require('lodash');
+
+export interface RatingsProps {
+    max: number;
+    rating: number | null;
+    Icon: React.StatelessComponent<IconWrapperProps>;
+    foregroundColor?: IconColor;
+    backgroundColor?: IconColor;
+}
+
+/**
+ * The ratings component shows the ratings score of an object.
+ * @param max - The maximum rating allowed (integer).
+ * @param rating - The actual rating of the object.
+ * @param Icon - The icon class to use. It should be wrapped by `IconWrapper`.
+ * @param foregroundColor - The color of the foreground icons. Defaults to 'orange'.
+ * @param backgroundColor - The color of the background icons. Defaults to 'grey'.
+ * @constructor
+ */
+export const Ratings: React.StatelessComponent<RatingsProps> = (
+    { max, rating, Icon, foregroundColor = "orange", backgroundColor = "grey" }
```

Comment:
for the other components we are using defaultProps to set the default. The benefit to this is that they then show in the info on storybook